calculation/intersection.py

```diff
@@ -337,15 +337,16 @@ def intersection_points_find(self, **kwargs):
                     ph1 = 180 - ph1 if ph1 > 90 else ph1
                     ph2 = 180 - ph2 if ph2 > 90 else ph2
```

Comment:
Isn't that supposed to be deleted?

Reply:
thanks, changed

Reply:
Mention not!

Reply:
> Mention not!

what did you try to mean here :)

Reply:
The main reason of the ph1 and ph2 to not look over 180° to 90° image.